Update the kong integration with log instruction (#2935)
DataDog/integrations-core: kong/README.md

        #    - instance:bar
    ```

See the [sample kong.d/conf.yaml][2] for all available configuration options.

2. [Restart the Agent][3] to begin sending Kong metrics to Datadog.

#### Log Collection

**Available for Agent >6.0**

> Kong access logs are actually generated by Nginx which is why the default location is the same one as nginx files.

Reviewer: ```suggestion
Kong access logs are generated by Nginx, so the default location is the same as for nginx files.
```

Reviewer 2: Additionally: all-caps NGINX.

> 1. Collecting logs is disabled by default in the Datadog Agent, you need to enable it in `datadog.yaml`:

Reviewer 2: ```suggestion
1. Collecting logs is disabled by default in the Datadog Agent. Enable it in `datadog.yaml`:
```
